pkgs/top-level/all-packages.nix

```diff
@@ -19348,7 +19348,10 @@ in
 
   wlroots = callPackage ../development/libraries/wlroots { };
 
-  sway = callPackage ../applications/window-managers/sway { };
+  sway-unwrapped = callPackage ../applications/window-managers/sway { };
+  sway = callPackage ../applications/window-managers/sway {
```

Comment:
I think what I meant was that `sway` would take `sway-unwrapped` as input/dependency and add a wrapper around `bin/sway`. This would have the advantage that one can customize the wrapper without having to rebuild Sway (it doesn't take very long, but I think it would be nice to avoid this).

I'm not totally sure but I think it would be best if we also move the wrapping for `swaybg` here and the functionality from the NixOS module as well (-> we would have only one wrapper instead of 2-3 and non-NixOS users could also use part of the functionality of the NixOS module, like the D-BUS part).